src/core/src/op/paged_attention.cpp

```diff
@@ -147,6 +147,35 @@ void PagedAttentionExtension::validate_and_infer_types() {
                           get_input_element_type(12),
                           ".");
 
+    NODE_VALIDATION_CHECK(
+            this,
+            get_input_partial_shape(13).rank().is_dynamic() ||
+            get_input_partial_shape(13).rank().get_length() == 0 ||
+            get_input_partial_shape(13).rank().get_length() == 1,
+            "Input `rotation_coefficients` should either have an empty shape or rank 1, but it has rank ",
+            get_input_partial_shape(13).rank().get_length(),
+            ".");
+    NODE_VALIDATION_CHECK(this,
+                          get_input_element_type(13).is_dynamic() || get_input_element_type(13) == element::f32,
+                          "Element type of `rotation_coefficients` input should be f32, but it is ",
+                          get_input_element_type(13),
+                          ".");
+
+    NODE_VALIDATION_CHECK(
+            this,
+            get_input_partial_shape(14).rank().is_dynamic() ||
+            get_input_partial_shape(14).rank().get_length() == 0 ||
+            get_input_partial_shape(14).rank().get_length() == 1,
+            "Input `rotated_block_indices` should either have an empty shape or rank 1 but it has rank ",
```

Comment:
The same comment are applicable here as for input 13 above.

Reply:
Done